### ISSUE TITLE
documentation + general confusion ;]

### ISSUE THREAD

**Issue by han5** [skipoleschris/xsbt-cucumber-plugin#32]
```
cucumber-jvm : 1.1.4 
scala : 2.10.0
sbt: 0.13
linux os: ubuntu 13.x / kernel 3.2.0-53-generic
jdk: 1.7
```

My project/plugins.sbt file: 

``` javascript
resolvers += "Local Maven Repository" at "file://"+Path.userHome.absolutePath+"/.m2/repository"

scalaVersion := "2.10.0"

addSbtPlugin("templemore" % "sbt-cucumber-integration" % "0.7.2" from "http://templemore.co.uk/repo/templemore/sbt-cucumber-integration_2.10/0.7.2/sbt-cucumber-integration_2.10-0.7.2.pom")

```

For the life of me, blew over +1hr of troubleshooting to find a resolver that would work, and finally gave up and used an explicit url.  

Any idea why the standard resolver entry fails (the one specified in nearly every document on "templemore" on the web)?  sbt injects all this extra info into the path which is not the "actual" path, must be ivy etc which I need to understand more.

So now with the above, it loads up in sbt and resolves success. 

But, just like the docs say, if I put this in my project.sbt file it will fail: 

``` javascript
name := "Cucumber Scala example"

version := "0.1"

scalaVersion := "2.10.0"

seq(cucumberSettings : _*)
```

The above fails with: [error] Type error in expression

And when trying the "sbt cucumber" command it fails with "cucumber not a valid key".

Can we just get 1 working simple sample with all the pieces in place in a folder tree for scala 2.10.0+ and sbt 0.12 or 0.13+ ? Especially with features-location specified and step-defiinitions-location specified, just load it up with settings for reference.

If I'm super newb then please point me/us to the proper info.

-thanks

**Comment by skipoleschris** [comment]
Looks like you have based your plugins.sbt file around one of my test projects which assumes a locally published version of the plugin. Please take a look at the main README.markdown file as this explains everything much more clearly. In particular, you need to use the following in the plugins.sbt file:

```
resolvers += "Templemore Repository" at "http://templemore.co.uk/repo"

addSbtPlugin("templemore" % "sbt-cucumber-plugin" % "0.8.0")
```

Everything else looks fine. Also note that I published a new 0.8.0 version today that supports the latest 1.1.4 version of cucumber jvm.

**Comment by han5** [comment]
Great that fixed it, thanks. :+1:
